packages/config/config/devices/0x001a/rf9501.json

```diff
@@ -103,6 +103,30 @@
 					"value": 2
 				}
 			]
+		},
+		{
+			"#": "10",
+			"label": "Notify Accessory",
+			"description": "Ensures that changes to the master node automatically notify accessory switches",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "DISABLE",
+					"value": 0
+				},
+				{
+					"label": "ENABLE",
+					"value": 1
+				}
+			]
 		}
-	]
+	],
+	"compat": {
+		"enableBasicSetMapping": true
+	}
```

Comment:
Can you explain why you added this?

Reply:
This was added because the control panel was not updating when i physically toggled the switch

Reply:
Makes sense, some other Eaton devices need this too.